projects.json

```diff
@@ -4782,7 +4789,23 @@
     "actions": [
       {
         "action": "BuildSwiftPackage",
-        "configuration": "release"
+        "configuration": "release",
+        "xfail": [
+          {
+            "issue": "rdar://124167314",
+            "compatibility": ["5.9"],
+            "branch": ["release/5.9"],
+            "job": ["source-compat"],
+            "platform": "Darwin"
```

Comment:
You can remove `platform` from this one and add a `"Only supports 5.10+"`. The underlying issue looks to be that these are run in the sandbox and the plugin itself is also run in a sandbox, which isn't supported.

Reply:
You beat me, your change looks good, thanks!